util/kvencoder/kv_encoder.go

```diff
@@ -226,6 +226,7 @@ func (e *kvEncoder) initial(dbName string, idAlloc autoid.Allocator) (err error)
 
 	se.GetSessionVars().IDAllocator = idAlloc
 	se.GetSessionVars().ImportingData = true
```

Comment:
Why set the default ImportingData to true?

Reply:
The session is created and maintained by the kvEncoder.